templates/taobao-creative-app/pages/index/index.js

```diff
@@ -0,0 +1,65 @@
+var touchstartCB;
+var touchcancelCB;
+var touchendCB;
+var touchmoveCB;
+
+function handleTouchEvent (event) {
+	if (my.isIDE) {
+		return;
+	}
+	let changedTouches = event.changedTouches;
+	if (changedTouches) {
+		for (let touch of changedTouches) {
+			touch.clientX = touch.x;
+			touch.clientY = touch.y;
+		}
+	}
+}
+
+Page({
+	onReady () {
+		my.onTouchStart = function (cb) {
```

Comment:
Like above, should move this function to unify.js? and in this file. only invoke my.dispatchEvent('touch-start'), my.dispatchEvent('touch-end')

Reply:
it's a little bit convoluted to implemented in `unify.js`

because `onShow` `onHide` hook is only available in app.js, 
and `onTouchXXX` hook is only available in page/index.js